.github/workflows/ci-packages.yml

```diff
@@ -104,4 +104,4 @@ jobs:
       # Test storybook build on packages that have changed (--include & --since)
       - name: Run storybook build for changed packages
         run: |
-          yarn workspaces foreach -tv --exclude '*-app' --exclude hobbies-helsinki --since=origin/main run build-storybook
+          yarn workspaces foreach -tv --exclude '*-app' --exclude hobbies-helsinki --exclude events-helsinki --since=origin/main run build-storybook
```

Comment:
exclude `*-helsinki` if we're using some naming pattern? Also `--exclude '*-app'` is probably not needed anymore as we dont have `nextjs-app` example anymore

Reply:
wops :D already merged. I'll do the changes in another PR

Reply:
fixed